package.json

```diff
@@ -3,14 +3,15 @@
     "node": "12.x"
   },
   "dependencies": {
-    "@rails/webpacker": "^5.2.1",
+    "@rails/webpacker": "^6.0.0-beta.5",
```

Comment:
We need to be sure this new version won't cause issues or conflicts with the Gov UK frontend lib.

Reply:
Reverted